config/migrations/2024/subsidies/20240315100754-update-lekp-2-1-deadlines.sparql

```diff
@@ -0,0 +1,452 @@
+PREFIX m8g:           <http://data.europa.eu/m8g/>
+PREFIX xsd:           <http://www.w3.org/2001/XMLSchema#>
+PREFIX dct:           <http://purl.org/dc/terms/>
+PREFIX mu:            <http://mu.semte.ch/vocabularies/core/>
+PREFIX subsidie:      <http://data.vlaanderen.be/ns/subsidie#>
+PREFIX mobiliteit:    <https://data.vlaanderen.be/ns/mobiliteit#>
+PREFIX lblodSubsidie: <http://lblod.data.gift/vocabularies/subsidie/>
+PREFIX qb:            <http://purl.org/linked-data/cube#>
+PREFIX xkos:          <http://rdf-vocabulary.ddialliance.org/xkos#>
+PREFIX cpsv:          <http://purl.org/vocab/cpsv#>
+PREFIX common:        <http://www.w3.org/2007/uwa/context/common.owl#>
+
+# Update LEKP 2.1 series to show '2023 – 2024' and 'N.V.T'
+DELETE {
+  GRAPH ?g {
+    ?s dct:title ?title .
+  }
+}
+INSERT {
+  GRAPH ?g {
+    ?s dct:title "2023 – 2024" .
+  }
+}
+WHERE {
+  GRAPH ?g {
+    VALUES ?s {
+      <http://data.lblod.info/id/subsidy-measure-offer-series/a04b604a-86c6-49c0-8b70-1910e3fdfa4c> # LEKP 2.1 series
+    }
+    ?s dct:title ?title .
+  }
+};
+
+# Remove from specific graphs
+INSERT DATA {
```

Comment:
Should this be `DELETE DATA` instead?

Reply:
Ah no my comment just ended up in the wrong place